js/start.js

```diff
@@ -38,6 +38,8 @@ import listingDeleteHandler from './startup/listingDelete';
 import { fixLinuxZoomIssue, handleLinks } from './startup';
 import ConnectionManagement from './views/modals/connectionManagement/ConnectionManagement';
 import Onboarding from './views/modals/onboarding/Onboarding';
+import SearchProvidersCol from './collections/search/SearchProviders';
+import getDefaultSearchProviders from './data/defaultSearchProviders';
```

Comment:
`getDefaultSearchProviders` doesn't make sense as the name for this anymore since you're not getting anything. It's already a list. I'd go with `defaultSearchProviders`.